spec/pragma.dd

```diff
@@ -302,7 +302,7 @@ $(H3 $(LNAME2 mangle, $(D pragma mangle)))
         extern(C++) struct _function {}
         template ScopeClass(C)
         {
-            pragma(mangle, C.stringof, C)
+            pragma(mangle, C)
```

Comment:
those args should be the other way around, not removed, see `compilable/issue21203.d`
```d
template ScopeClass(C , string name = C.stringof)
//if (is(C == class) && __traits(getLinkage, C) == "C++")
{
    //enum name = C.stringof;
    enum ns = __traits(getCppNamespaces,C);
    extern(C++, class)
    {
        extern(C++,(ns))
        {
            pragma(mangle, C, name)
            struct ScopeClass
            {
                char[__traits(classInstanceSize, C)] buffer;
                //... all the things ...
            }
        }
    }
}
```